docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -5265,6 +5265,18 @@ The `swift_storage_backend` block configures the connection to OpenStack Object
 &nbsp;
 
 ```yaml
+# OpenStack Swift application credential id
```

Comment:
```suggestion
# OpenStack Swift application credential ID
```

Reply:
done!

Reply:
Sorry, this should have been updated on the CLI flag which is the source for this generated documentation. After doing that, please run `make docs` and `make reference-help` again and commit the results.

Reply:
ok did that